robot-core/src/main/java/org/obolibrary/robot/ReasonerHelper.java

```diff
@@ -225,6 +229,66 @@ private static OWLOntology createIncoherentModule(
     return module;
   }
 
+  /**
+   * Given a list of axiom generator strings, return a list of InferredAxiomGenerator objects.
+   *
+   * @param axGenerators list of strings to get InferredAxiomGenerators
+   * @return list of InferredAxiomGenerators
+   * @throws Exception on problem getting InferredAxiomGenerator
+   */
+  public static List<InferredAxiomGenerator<? extends OWLAxiom>> getInferredAxiomGenerators(
+      List<String> axGenerators) throws Exception {
+    List<InferredAxiomGenerator<? extends OWLAxiom>> gens = new ArrayList<>();
+    if (axGenerators == null || axGenerators.isEmpty()) {
+      gens.add(new InferredSubClassAxiomGenerator());
+      return gens;
+    }
+    for (String ax : axGenerators) {
+      gens.add(getInferredAxiomGenerator(ax));
+    }
+    return gens;
+  }
+
+  /**
+   * Given an axiom generator as a string, return the InferredAxiomGenerator object.
+   *
+   * @param axGenerator name of InferredAxiomGenerator
+   * @return InferredAxiomGenerator
+   * @throws Exception if the string is not a valid axiom generator
+   */
+  public static InferredAxiomGenerator<? extends OWLAxiom> getInferredAxiomGenerator(
+      String axGenerator) throws Exception {
+    switch (axGenerator.toLowerCase()) {
+      case "subclass":
+      case "":
+        return new InferredSubClassAxiomGenerator();
+      case "disjointclasses":
+        return new InferredDisjointClassesAxiomGenerator();
+      case "equivalentclass":
+        return new InferredEquivalentClassAxiomGenerator();
+      case "datapropertycharacteristic":
+        return new InferredDataPropertyCharacteristicAxiomGenerator();
+      case "equivalentdataproperty":
```

Comment:
For consistency, I think this should be "properties" plural to match the class name.

Reply:
I was debating that one, since (oddly) the corresponding object property class is `InferredEquivalentObjectPropertyAxiomGenerator` while the data property one uses "properties". I guess as long as the correct one shows up in the docs, it will be clear.

Reply:
Ok. I'd rather stick to the OWLAPI names. Either they have good reasons, or we can blame inconsistencies on them. :)

Reply:
Fixed!